### DISCUSSION TITLE
bug: settings "number of most relevant contact entries" has no effect

### DISCUSSION THREAD

**Discussion by drfedtke** [devmount/third-stats#373] [General]
hi,

you made great work with your project.

one bug I would like to report: the top-n setting does not work with large numbers:

![grafik](https://user-images.githubusercontent.com/9048017/160224600-39c26375-07f9-49dd-a8c3-9d697a4ce211.png)

![grafik](https://user-images.githubusercontent.com/9048017/160224529-ea19336e-7d10-4b6d-ac7d-a0c03aee6519.png)

does still result in just a top-20:

![grafik](https://user-images.githubusercontent.com/9048017/160224510-e4ed2473-5f05-474c-b691-c4d49984d5d6.png)

could this be fixed? good is that there is space below that 
dashboard element.

best regards
stephen

**Comment by devmount** [comment]
Hi Stephen,
I'm glad you like this add-on and thanks for this report.

As you can see in your last screenshot: there are indeed lots of entries (veeeery thin blue bars in the bar chart), only the labels in between are skipped so that the text won't become unreadable due to overlapping. When hovering over the thin bars, you should see the corresponding tooltips.

I know that this isn't the most accessible visualization. I'll take a look at how to improve this chart (maybe as you suggested with a kind of vertical expander).

**Comment by drfedtke** [comment]
hi andreas,

thanks for your reply. great.

yes, something like a "show all" button would help, and maybe also a 
"copy" button
to copy this top-xxx email id list to the clipboard.

best regards, thanks
stephen (fedtke)

**Reply by devmount** [reply]
I created #374 for this. This maybe won't make it into the next release, but the one after I hope. Thanks again for the report.

**Reply by devmount** [reply]
@drfedtke Copying individual entries to clipboard is now implemented in #508 and will be part of the next release. To have the whole list, you can also always use the json export in the top right corner.
